core/transformer/src/IModelTransformer.ts

```diff
@@ -744,7 +744,22 @@ export class IModelTransformer extends IModelExportHandler {
   /** Override of [IModelExportHandler.shouldExportElementAspect]($transformer) that is called to determine if an ElementAspect should be exported from the source iModel.
    * @note Reaching this point means that the ElementAspect has passed the standard exclusion checks in [IModelExporter]($transformer).
    */
-  protected override shouldExportElementAspect(_sourceAspect: ElementAspect): boolean { return true; }
+  protected override shouldExportElementAspect(sourceAspect: ElementAspect): boolean {
+    // When exporting elements, the transformer may choose to *defer* an element, which is something the exporter doesn't understand,
+    // so it tries to export aspects anyway. As such we need to *not* export aspects for deferred elements.
+    // When we retry processing deferred elements and successfully export them is when we export the aspects again
+    const targetElementId = this.context.findTargetElementId(sourceAspect.element.id);
+    if (targetElementId === Id64.invalid && this._deferredElementIds.has(sourceAspect.element.id)) {
+      Logger.logWarning(
```

Comment:
should this really be a warning?  It seems like this will happen durning normal operation and cause a lot of warnings.

Reply:
you're right, I will move it to logInfo